test/int/fixtures/launchWebServer.ts

```diff
@@ -65,4 +65,14 @@ export class LaunchWebServer implements IFixture {
     public toString(): string {
         return `LaunchWebServer`;
     }
+}
+
+export class ProvideStaticUrl implements IFixture {
```

Comment:
I'd either move this to a different file, or rename this file to some other name that encompass both launch web server + provide static url.